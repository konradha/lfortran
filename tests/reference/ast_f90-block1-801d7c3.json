{
    "basename": "ast_f90-block1-801d7c3",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/block1.f90",
    "infile_hash": "f96fe7a8f6b65ebcf0a50e8d6fa12979410f5f902e0771f5100aaad1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-block1-801d7c3.stdout",
    "stdout_hash": "d3d4d4938156b0f13e504bf0a9385d14fd65949002457578b26b5456",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}