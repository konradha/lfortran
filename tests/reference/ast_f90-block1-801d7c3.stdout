program block1
implicit integer (a-z)
integer :: B
block
    use mod, only: example
    import, none
!import, only: B
    B = 10
end block
end program block1
! The variable B is implicitly declared in the scoping unit of the main program.
! The statement IMPORT, NONE makes B inaccessible in the BLOCK construct.
! If the IMPORT, NONE statement is replaced with the IMPORT statement in the
! comment, the program is conformant.
